Add prerequisites info to attached cluster resource reference doc (#883)
GoogleCloudPlatform/k8s-config-connector: scripts/generate-google3-docs/resource-reference/generated/resource-docs/containerattached/containerattachedcluster.md

</tbody>
</table>

## Prerequisites
We need to prepare the target cluster to be attached to make sure the Multi-cloud service connects to the target cluster.
That requires an install-agent to be deployed into the target cluster.

Steps to deploy the install-agent into target cluster:

1. Get the manifest yaml file for the install-agent

`gcloud container attached clusters generate-install-manifest my-cluster  --location=GOOGLE_CLOUD_REGION --platform-version=PLATFORM_VERSION --output-file=manifest.yaml`

Example command:

`gcloud container attached clusters generate-install-manifest kcc-attached-cluster  --location=us-west1 --platform-version=1.25.0-gke.5 --output-file=manifest.yaml`

2. Checkout the target cluster and get kubeconfig context

EKS cluster: `aws eks update-kubeconfig --region $AWS_REGION --name $CLUSTER`

AKS cluster: `az aks get-credentials -n $CLUSTER -g $AZURE_RESOURCE_GROUP`

`export KUBECONFIG_CONTEXT=$(kubectl config current-context)`

3. Apply manifest yaml to target cluster

> `kubectl use-context $KUBECONFIG_CONTEXT` (optional, previous aws/az command should already switch context)

Author: I prefer move it to the beginning, still want to leave this command here for future reference.

</tbody>
</table>

## Prerequisites
Before you can use this resource, you must prepare the target cluster
so that the multi-cloud service can connect to the target cluster.
To prepare the cluster, follow the steps to deploy an install-agent into the target cluster:

> Steps to deploy the install-agent into target cluster:

Reviewer: remove this line since we moved the introduction into the previous paragraph

1. Get the manifest yaml file for the install-agent:

`gcloud container attached clusters generate-install-manifest my-cluster  --location=GOOGLE_CLOUD_REGION --platform-version=PLATFORM_VERSION --output-file=manifest.yaml`

Example command:

`gcloud container attached clusters generate-install-manifest kcc-attached-cluster  --location=us-west1 --platform-version=1.25.0-gke.5 --output-file=manifest.yaml`

2. Check out the target cluster and get the kubeconfig context:

EKS cluster: `aws eks update-kubeconfig --region $AWS_REGION --name $CLUSTER`

AKS cluster: `az aks get-credentials -n $CLUSTER -g $AZURE_RESOURCE_GROUP`

`export KUBECONFIG_CONTEXT=$(kubectl config current-context)`

3. Apply the `manifest.yaml` file to the target cluster:

> (optional, previous aws/az command should already switch context) `kubectl use-context $KUBECONFIG_CONTEXT`

Reviewer: ```suggestion
(Optional if you used the previous command to switch context) `kubectl use-context $KUBECONFIG_CONTEXT`
```
